nbeats_keras/model.py

```diff
@@ -126,19 +171,19 @@ def wrapper(*args, **kwargs):
 
 
 def linear_space(backcast_length, forecast_length, fwd_looking=True):
-    ls = K.arange(-float(backcast_length), float(forecast_length), 1) / backcast_length
+    l = K.arange(-float(backcast_length), float(forecast_length), 1) / backcast_length
```

Comment:
PEP8: Ambiguous variable name: 'l'. I don't know why l is ambiguous so that's why I renamed it to ls.

Reply:
I do used l as index in a for loop...
Updated to ls